src/sentry/tasks/post_process.py

```diff
@@ -422,7 +422,25 @@ def run_post_process_job(job: PostProcessJob):
     for pipeline_step in pipeline:
         try:
             pipeline_step(job)
+            logger.info(
+                "post_process.pipeline_step",
+                extra={
```

Comment:
Optional: You may want to create an extra variable with some of the values pre-filled and then just update what you need before calling logger.

Something like this:
```python
extra = {"event": event.event_id, "group": event.group_id}
logger.info("foo", extra=extra.update({"status": "started"})
```